### PR TITLE
(port) permabrig license plate press

### PR DESCRIPTION
**Pull request by boodaliboo**
original pr: https://github.com/tgstation/tgstation/pull/49452
"adds a license press prisoners can use to make license plates from empty ones. Makes a small profit"
#### Changelog

:cl:  
rscadd: Added a new kinda job for permabrig prisoners to make license plates
/:cl:

### CODE REVIEW

**Comment by boodaliboo** [comment]
image
 this needs defining

**Comment by UselessTheremin** [comment]
....what are license plates for

**Comment by boodaliboo** [comment]
> ....what are license plates for

selling

**Comment by UselessTheremin** [comment]
> > ....what are license plates for
> 
> selling

ah ok cool

**Comment by Tipy1802** [comment]
very nice, would make more sense if it was a workcamp thing though. Maybe even have the blank plates be made from metal.
But even in this form this is very cool

**Comment by boodaliboo** [comment]
i need to either define the var or find a new one

**Review by patpol4 — CHANGES_REQUESTED** [review]
Don't place the machine in the pathway, you're preventing people from walking, try the left or right side of perma maybe?

**Review by JamieD1 — APPROVED** [review]
Approving ART ONLY for now.

File: code/game/objects/items/stacks/license_plates.dm [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+/obj/item/stack/license_plates/empty/fifty
+	amount = 50
+
+/obj/item/stack/license_plates/filled
+	name = "license plate"
+	desc = "Prison labor paying off."
+	icon_state = "filled_plate_1_1"
+
+///Override to allow for variations
+/obj/item/stack/license_plates/filled/update_icon_state()
```
Comment (Bibby0110) [inline code/game/objects/items/stacks/license_plates.dm]:
```suggestion
/obj/item/stack/license_plates/filled/update_icon()
```
pretty sure

**Comment by Marmio64** [comment]
once again, another broken PR by bood

**Comment by Bibby0110** [comment]
> once again, another broken PR by bood

shut the fuck up

**Comment by Xoxeyos** [comment]
How do these perma prisoners turn in their cash? Their ID's aren't exactly with them in perma. TG has a prisoner role, and I don't believe this PR is necessary for our server. https://tgstation13.org/wiki/Prisoner

**Comment by boodaliboo** [comment]
> How do these perma prisoners turn in their cash? Their ID's aren't exactly with them in perma. TG has a prisoner role, and I don't believe this PR is necessary for our server. https://tgstation13.org/wiki/Prisoner

cash is useless for prisoners but sec can sell them

**Comment by patpol4** [comment]
Reminder this one is still waiting on a map change

**Pull request closed without merging**